.changeset/nervous-games-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
```

Comment:
Can we update this to `minor` please, given the prop name has changed. 

Also can an example be provided here to explain the migration plan please?

something like:

**Before:** 
```jsx
<Card label="Read more">
...
</Card>
```

**after:** 
```jsx
<Card ctaText="Read more">
...
</Card>
```